app/controllers/api/base_controller.rb

```diff
@@ -9,6 +9,57 @@ class BaseController < ActionController::API
     rescue_from ActionController::BadRequest, with: :bad_request_response
     rescue_from ArgumentError, with: :bad_request_response
 
+    def append_info_to_payload(payload)
+      super
+      payload[:current_user_class] = current_lead_provider&.class&.name
+      payload[:current_user_id] = current_lead_provider&.id
+      payload[:current_user_name] = current_lead_provider&.name
+
+      payload[:request_headers] = request.env.slice(
```

Comment:
Why do we need this list of headers? Wouldn't be better to add all headers regardless or just remove some like `HTTP_AUTHORIZATION`?

Reply:
I'll remove auth